### PR TITLE
Gabriel Calero's fix.

### PR DESCRIPTION
**Pull request by NissimHadar**
# Description
Fixes broken Android build

### CODE REVIEW

File: android/docker/Dockerfile [hunk 1]
```diff
@@ -26,7 +26,7 @@ RUN mkdir -p "$ANDROID_HOME" "$ANDROID_SDK_HOME" && \
     curl -s -S -o sdk.zip -L "${SDK_URL}" && \
     unzip sdk.zip && \
     rm sdk.zip && \
-    yes | $ANDROID_HOME/tools/bin/sdkmanager --licenses
+    yes | $ANDROID_HOME/tools/bin/sdkmanager --licenses && yes | $ANDROID_HOME/tools/bin/sdkmanager --update
 
 # Install Android Build Tool and Libraries
 RUN $ANDROID_HOME/tools/bin/sdkmanager --update
```
Comment (SamGondelman) [inline android/docker/Dockerfile]: still happening twice?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16615/HighFidelity-Beta-PR14751-dd2a824.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16615/HighFidelity-Beta-PR14751-dd2a824.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16615/HighFidelity-Beta-Interface-PR14751-dd2a824.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16615/HighFidelity-Beta-PR14751-dd2a824.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16615/HighFidelity-Beta-Interface-PR14751-dd2a824.exe

**Comment by jherico** [comment]
this PR is targeting the wrong branch.

File: android/docker/Dockerfile [hunk 2]
```diff
@@ -26,10 +26,9 @@ RUN mkdir -p "$ANDROID_HOME" "$ANDROID_SDK_HOME" && \
     curl -s -S -o sdk.zip -L "${SDK_URL}" && \
     unzip sdk.zip && \
     rm sdk.zip && \
-    yes | $ANDROID_HOME/tools/bin/sdkmanager --licenses
+    yes | $ANDROID_HOME/tools/bin/sdkmanager --licenses && yes | $ANDROID_HOME/tools/bin/sdkmanager --update
```
Comment (jherico) [inline android/docker/Dockerfile]:
```suggestion
    yes | $ANDROID_HOME/tools/bin/sdkmanager --licenses && \ 
    yes | $ANDROID_HOME/tools/bin/sdkmanager --update
```

**Comment by conklin94122** [comment]
@NissimHadar I'm closing this PR. We need it based on the branch "v0.78.0-rc". We already have a PR against master, https://github.com/highfidelity/hifi/pull/14750

**Pull request closed without merging**